Doc/tools/version_next.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+"""
+Replace `.. versionchanged:: next` lines in docs files by the given version.
+
+Run this at release time to replace `next` with the just-released version
+in the sources.
+
+No backups are made; add/commit to Git before running the script.
+
+Applies to all the VersionChange directives. For deprecated-removed, only
+handle the first argument (deprecation version, not the removal version).
+
+"""
+
+import re
+import sys
+import argparse
+from pathlib import Path
+
+DIRECTIVE_RE = re.compile(
+    r'''
+        (?P<before>
+            \s*\.\.\s+
+            (version(added|changed|removed)|deprecated(-removed)?)
+            \s*::\s*
+        )
+        next
+        (?P<after>
+            .*
+        )
+    ''',
+    re.VERBOSE | re.DOTALL,
+)
+
+docdir = (Path(__file__)
+          .parent  # cpython/Doc/tools
+          .parent  # cpython/Doc
+          .resolve()
+          )
+
+parser = argparse.ArgumentParser(
+    description=__doc__,
+    formatter_class=argparse.RawDescriptionHelpFormatter)
+parser.add_argument('version',
+                    help='String to replace "next" with.')
```

Comment:
Maybe something like this, to remind we want a version like 3.14 not 3.14.0 or 3.14.0a0?
```suggestion
                    help='"x.y" version string to replace "next" with.')
```

Reply:
I'd rather keep the tool generic, and pass (or enforce?) `x.y` in the script that calls it (`run_release.py`).

Reply:
So I went with `String to replace "next" with. Usually `x.y`, but can be anything.`